content/pt-br/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,135 @@
+
+Nem todos os logs estarão garantidamente em formato JSON (como por exemplo durante o início de processos). Se você pretender realizar um parsing dos logs, seu código deverá saber tratar também linhas que não são JSON
+
+O nome dos campos e a serialização JSON ainda estão sujeitos a mudanças.
+{{< /warning >}}
+
+A opção `--logging-format=json` muda o formato dos logs, do formato padrão da klog para JSON. Abaixo segue um exemplo de um log em formato JSON (identado):
+```json
+{
+   "ts": 1580306777.04728,
+   "v": 4,
+   "msg": "Pod status updated",
+   "pod":{
+      "name": "nginx-1",
+      "namespace": "default"
+   },
+   "status": "ready"
+}
+```
+
+Chaves com significados especiais:
+* `ts` - Data e hora no formato Unix (obrigatório, float)
+* `v` - Nível de detalhe (obrigatório, int, padrão 0)
+* `err` - Mensagem de erro (opcional, string)
+* `msg` - Mensagem (obrigatório, string)
+
+Lista dos componentes que suportam o formato JSON atualmente:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+* {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}
+* {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
+* {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+
+### Limpeza dos Logs
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+{{<warning >}}
+A funcionalidade de limpeza dos logs pode causar impactos significativos na performance, sendo portanto contraindicada em produção.
+{{< /warning >}}
+
+A opção `--experimental-logging-sanitization` habilita o filtro de limpeza dos logs.
+Quando habilitado, esse filtro inspeciona todos os argumentos dos logs, procurando por campos contendo dados sensíveis (como senhas, chaves e tokens). Tais campos não serão expostos nas mensagens de log.
+
+Lista dos componentes que suportam a limpeza de logs atualmente:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+* {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}
+* {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
+* {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+
+{{< note >}}
+O filtro de limpeza dos logs não impede a exposição de dados sensíveis nos logs das aplicações em execução.
+{{< /note >}}
+
+### Nível de detalhe dos logs
+
+A opção `-v` controla o nível de detalhe dos logs. Um valor maior aumenta o número de eventos registrados, começando a registrar também os eventos menos importantes. Similarmente, um valor menor restringe os logs apenas aos eventos mais importantes. O valor padrão 0 registra apenas eventos críticos.
+
+### Localização dos Logs
+
+Existem dois tipos de componentes do sistema: aqueles que são executados em um container e aqueles que não são. Por exemplo:
```

Comment:
container to contêiner

Reply:
Bad custom of mixing foreign names 😅. Fixed all of them.